Homeworks/Python/Seminar_1/Task_6.py

```diff
@@ -0,0 +1,17 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с 
+# номером. Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме 
+# последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, 
+# которая проверяет счастливость билета.
+# *Пример:*
+
+# 385916 -> yes
+# 123456 -> no
+
+number = int(input('Vvedite nomer bileta, sostoyaschiy iz 6 chisel: '))
+if number >= 100000 and number < 1000000:
+    if ((number % 10) + (number % 100 // 10) + (number % 1000 // 100)) == ((number % 10000 // 1000) + (number % 100000 // 10000) + (number % 1000000 // 100000)):
```

Comment:
Тут нигде скобки не нужны.
```suggestion
    if number % 10) + number % 100 // 10 + number % 1000 // 100 == number % 10000 // 1000 + number % 100000 // 10000 + number % 1000000 // 100000:
```